### DISCUSSION TITLE
Workflows can break if you ran "Update System Files" against a specific branch

### DISCUSSION THREAD

**Discussion by jonaswre** [microsoft/AL-Go#1585] [Ideas]
I just realised that you can break your workflows if you "Update System Files" to a branch and then delete that branch. 

You wont be able to run any workflows anymore. 

https://github.com/byndit-demo/App-B/actions/runs/8137886646

The workaround here is to copy the workflows from another repo and update these locally.

I just thought it might be useful to document that somewhere.

**Comment by freddydk** [comment]
The easiest workaround is to recreate the branch and then run update to something else.

**Comment by jonaswre** [comment]
It's a branch on your repo

**Comment by freddydk** [comment]
Which I can easily recreate if you like.
Typically you would only use branches like this for testing a fix (instructed by us) - and if the branch is deleted, we can recreate it.
But... - you are right, we should document this when we instruct people in testing a fix.

**Comment by freddydk** [comment]
And... - maybe I should add this to the troubleshooting workflow - test whether the actions branch exist - and describe how to fix it - probably a good idea.
